kevm-pyk/src/kevm_pyk/foundry.py

```diff
@@ -391,8 +402,29 @@ def foundry_prove(
     setup_methods: dict[str, str] = {}
     contracts = unique({test.split('.')[0] for test in tests})
     for contract_name in contracts:
-        if 'setUp' in foundry.contracts[contract_name].method_by_name:
-            setup_methods[contract_name] = f'{contract_name}.setUp'
+        contract = foundry.contracts[contract_name]
+        if 'setUp' in contract.method_by_name:
+            setup_method = contract.method_by_name['setUp']
+            setup_methods[contract_name] = f'{setup_method.contract_name}.{setup_method.name}'
```

Comment:
Why was this change made? It doesn't seem to compute anything different...

Reply:
I might have been trying to isolate a crash. It's changed back now.